.github/workflows/toxicity-check.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Toxicity check'
+on: 
+  issue_comment:
+    types: ['created', 'edited', 'deleted']
+  pull_request_review:
+  pull_request_review_comment:
+  issues:
+    types: ['opened', 'edited', 'deleted', 'closed', 'reopened', 'locked', 'unlocked']
```

Comment:
I am very curious so asking Just for knowledge, what/how will you detect on issue closed and reopened? 

Same for pull request too...

Reply:
Never mind, I was so curious what are you doing on `closed` and `reopened` **for toxicity** and found myself that you are not doing anything 😉  (just adding logs)
https://github.com/TheDutchDevil/toxicity-detection-bot/blob/main/eventProcessor.ts#L146